rmarkdowns/Week_2.Rmd

```diff
@@ -189,6 +227,47 @@ ggplot() +
 
 ```
 
+
+```{r}
+populated_places <- ne_download(
+  scale = 50,
+  type = "populated_places"
+)
+```
+
+
+```{r}
+
+# Download the protected areas layer
+reefs <- ne_download(
```

Comment:
The reefs data is only available at the scale = 10, so if you add that line to your call, it should work